ci/Makefile

```diff
@@ -49,30 +59,25 @@ restart-batch-proxy:
                            -o name \
                          | sed 's:pods/::' \
                          | head -n 1))
-	kubectl port-forward ${BATCH_POD} 8888:5000 > batch-proxy.log 2>batch-proxy.err & echo $$! > batch-proxy.pid
-
-
+	kubectl port-forward $(BATCH_POD) 8888:5000 > batch-proxy.log 2>batch-proxy.err & echo $$! > batch-proxy.pid
 
 run-local: HAIL_CI_REMOTE_PORT = 3000
-run-local: PROXY_IP=$(shell gcloud compute instances list --filter=name=dk-test --format="value(EXTERNAL_IP)")
 run-local: restart-all-proxies
-	SELF_HOSTNAME=http://${PROXY_IP}:${HAIL_CI_REMOTE_PORT} \
-	BATCH_SERVER_URL=http://127.0.0.1:${HAIL_CI_LOCAL_BATCH_PORT} \
+	SELF_HOSTNAME=http://$(EXTERNAL_IP):$(HAIL_CI_REMOTE_PORT) \
+	BATCH_SERVER_URL=http://127.0.0.1:$(HAIL_CI_LOCAL_BATCH_PORT) \
 	source activate hail-ci && python run_ci.py
 
 run-local-for-tests: HAIL_CI_REMOTE_PORT = 3001
-run-local-for-tests: PROXY_IP=$(shell gcloud compute instances list --filter=name=dk-test --format="value(EXTERNAL_IP)")
 run-local-for-tests: restart-all-proxies
-	SELF_HOSTNAME=http://${PROXY_IP}:${HAIL_CI_REMOTE_PORT} \
-	BATCH_SERVER_URL=http://127.0.0.1:${HAIL_CI_LOCAL_BATCH_PORT} \
+	SELF_HOSTNAME=http://$(EXTERNAL_IP):$(HAIL_CI_REMOTE_PORT) \
+	BATCH_SERVER_URL=http://127.0.0.1:$(HAIL_CI_LOCAL_BATCH_PORT) \
 	WATCHED_TARGETS='[["hail-is/ci-test:master", true]]' \
-	source activate hail-ci && pip install ./batch && python run_ci.py
+	source activate hail-ci && pip install ../batch && python run_ci.py
```

Comment:
want to double check that this is correct; we're running this from /ci, not / right?

Reply:
yes